src/libstd/ffi/mod.rs

```diff
@@ -8,7 +8,145 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Utilities related to FFI bindings.
+//! This module provides utilities to handle C-like strings.  It is
```

Comment:
I think that this is a bit misleading because `OsString` isn't a C string on Windows.

Reply:
A better way to describe it might be "to handle data across non-Rust interfaces, like other programming languages and the underlying operating system"